packages/block-library/src/tag-cloud/index.php

```diff
@@ -15,9 +15,12 @@
 function render_block_core_tag_cloud( $attributes ) {
 	$args      = array(
 		'echo'       => false,
+		'unit'       => substr( $attributes['smallestFontSize'], -2 ),
```

Comment:
Hi thank you for the comment, could you please elaborate a bit more. I didn't get it properly.

Reply:
Since we are going this way to allow more `units`, we shouldn't restrict it to `pt and px` only. So for example if we have `rem`, the `slice( -2 )` would not work as expected. Check some usages of `useCustomUnits` where `useSetting` is also used. Example:
```
const units = useCustomUnits( {
	availableUnits: useSetting( 'spacing.units' ) || [ 'px', 'em', 'rem' ],
} );
```

Reply:
We are using `useCustomUnits` hook!